assembly_instructions/readme.md

```diff
@@ -0,0 +1,30 @@
+Prior to starting the assembly make sure that all the items from the list below are present.
+
+* Appropriate motor (for actual motor model check readme in the root of this repository )
+* Thermistor with pre-soldered extension wires
+* A set of PCBs:
+    * Connectors PCB
+    * Logic PCB
+    * Power stage PCB
+* Housing:
+    * Top part
+    * Bottom part
+* Thermal pad
+* A set of screws for the motor
+* A set of screws for the housing itself
+
+  ![](BOM.png)
```

Comment:
```suggestion
![](BOM.png)
```

Please, do not indent images because it breaks the formatting. In this case, the extra indentation makes it part of the list, which is why it is rendered with large vertical whitespace.

Reply:
*